Gui.py

```diff
@@ -39,8 +43,9 @@ def initUI(self):
         self.setWindowIcon(QIcon('asset/img/groot.png'))
 
         #Function of speaker button linked:
-        speaker.clicked.connect(listen) #When the Speak button is clicked the listen function
-        #is triggered.
+        speaker.clicked.connect(listen)
+        #When the Speak button is clicked the listen function is triggered.
```

Comment:
This should go above the line number `46` because this is for that line. :thinking: